src/Leela.cpp

```diff
@@ -207,11 +207,7 @@ static void parse_commandline(int argc, char *argv[]) {
 
     if (!vm["threads"].defaulted()) {
         auto num_threads = vm["threads"].as<int>();
-        if (num_threads > cfg_max_threads) {
-            myprintf("Clamping threads to maximum = %d\n", cfg_max_threads);
-        } else if (num_threads != cfg_num_threads) {
-            cfg_num_threads = num_threads;
-        }
+        cfg_num_threads = num_threads;
```

Comment:
This needs to respect cfg_max_threads somehow or you risk crashing in OpenBLAS. Yanking up cfg_max_threads to ```std::max(1, std::min(SMP::get_num_cpus() * gpus, MAX_CPUS)); maybe?```

Reply:
No objection.  I don't think I am qualified enough to make a call here since I am not aware of how OpenBLAS crashes.  Will push an update soon.

Reply:
FYI will need a bit of changes (not too complicated though) simply because cfg_max_threads were originally calculated before we parsed the GPU options.